website/docs/r/emr_cluster.html.markdown

```diff
@@ -0,0 +1,104 @@
+The following arguments are supported:
+
+* `name` - (Required) The name of emr cluster.
+* `emr_ver` - (Required, ForceNew) EMR Version, e.g. EMR-3.22.0. 
+* `cluster_type` - (Required, ForceNew) EMR Cluster Type, e.g. HADOOP, KAFKA, DRUID, etc.
+* `zone_id` - (Required, ForceNew) Zone ID.
+* `security_group_id` (Optional, ForceNew) Security Group ID for Cluster, you can also specify this key for each host group.
+* `security_group_name` (Optional, ForceNew) Security Group Name.
+* `net_type` (Required, ForceNew) Net type, e.g. vpc.
+* `vpc_id` (Required, ForceNew) Vpc id.
+* `vswitch_id` (Optional, ForceNew) Global vswitch id, you can also specify it in host group.
+* `option_software_list` (Optional, ForceNew) Optional software list.
+* `high_availability_enable` (Optional, ForceNew) High Available for HDFS and YARN. If this is set true, MASTER group must have two nodes.
+* `use_local_metadb` (Optional, ForceNew) Use local metadb. Default is false.
+* `io_optimized` (Optional, ForceNew) Whether ECS instance is io-optimized. Default value is true.
+* `ssh_enable` (Optional, ForceNew) If this is set true, we can ssh into cluster.
+* `master_pwd` (Optional, ForceNew) Master ssh password.
+* `eas_enable` (Optional, ForceNew) High security cluster (true) or not.
+* `user_defined_emr_ecs_role` (Optional, ForceNew) Alicloud EMR uses roles to perform actions on your behalf when provisioning cluster resources, running applications, dynamically scaling resources. EMR uses the following roles when interacting with other Alicloud services. Default value is AliyunEmrEcsDefaultRole.
+* `key_pair_name` (Optional, ForceNew) Ssh key pair.
+* `deposit_type` (Optional, ForceNew) Cluster deposit type, HALF_MANAGED or FULL_MANAGED.
+* `related_cluster_id` (Optional, ForceNew) This specify the related cluster id, if this cluster is a Gateway.
+* `host_group` - (Required, ForceNew) Groups of Host, You can specify MASTER as a group, CORE as a group (just like the above example).
+  - `host_group_name` - (Optional) host group name.
+  - `host_group_type` - (Required) host group type, MASTER, CORE or TASK.
+  - `charge_type` - (Optional) Charge Type for this group of hosts: PostPaid or PrePaid. If this is not specified, charge type will follow outside charge_type value.
+  - `period` - (Optional) If charge type is PrePaid, this should be specified, unit is month.
+  - `node_count` - (Required) Host number in this group.
+  - `instance_type` - (Required) Host Ecs instance type.
+  - `disk_type` - (Required) Data disk type.
+  - `disk_capacity` - (Required) Data disk capacity.
+  - `disk_count` - (Required) Data disk count.
+  - `sys_disk_type` - (Required) System disk type.
+  - `sys_disk_capacity` - (Required) System disk capacity.
+  - `auto_renew` - (Optional) Auto renew for prepaid, true of false.
+  - `vswitch_id` - (Optional) Vswitch id for this Group of hosts.
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `id` - The cluster ID.
+
```

Comment:
If the resource supports `Importer`, please show it in the docs.

Reply:
not support

Reply:
> If the resource supports `Importer`, please show it in the docs.

not support